priv/repo/migrations/20170317153551_create_message.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Healthlocker.Repo.Migrations.CreateMessage do
+  use Ecto.Migration
+
+  def change do
+    create table(:messages) do
+      add :body, :string, null: false
+      add :name, :string
```

Comment:
Any reason we need to copy the user's name into the `messages` table, rather than reading it from the `users` tables? Kinda goes against database normalization? Will start getting tricky if users update their names etc.